### PR TITLE
Expose preconfigured mobile device emulation settings

### PR DESCRIPTION
**Pull request by imiric**
This enables the previously commented out device emulation settings list. See the example for how to use it.

It's related to #113, which turned out to not be an issue, but this makes the configuration user friendlier.

### CODE REVIEW

**Review by olegbespalov — APPROVED** [review]
:+1:

File: common/browser_context_options.go [hunk 1]
```diff
@@ -53,24 +53,14 @@ type BrowserContextOptions struct {
...
-		Geolocation:       nil,
-		HasTouch:          false,
-		HttpCredentials:   nil,
-		IgnoreHTTPSErrors: false,
-		IsMobile:          false,
 		JavaScriptEnabled: true,
 		Locale:            DefaultLocale,
-		Offline:           false,
-		Permissions:       make([]string, 0),
+		Permissions:       []string{},
```
Comment (inancgumus) [inline common/browser_context_options.go:61]: You can remove this too.

**Review by inancgumus — COMMENTED** [review]
LGTM.

File: common/device.go [hunk 2]
```diff
@@ -22,1006 +22,797 @@ package common
...
-				DeviceScaleFactor: 1,
-				IsMobile:          true,
-				HasTouch:          true,
-				IsLandscape:       true,
+				Width:  1024,
+				Height: 600,
 			},
+			DeviceScaleFactor: 1,
+			IsMobile:          true,
+			HasTouch:          true,
```
[the remark this answers is not in the document]
Comment (imiric) [reply common/device.go:56]: Right. I removed it because it can be inferred from the width and height.

**Pull request merged**